pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -321,6 +321,9 @@
     public static final String CONFIG_OF_ENABLE_THREAD_CPU_TIME_MEASUREMENT =
         "pinot.server.instance.enableThreadCpuTimeMeasurement";
     public static final boolean DEFAULT_ENABLE_THREAD_CPU_TIME_MEASUREMENT = false;
+
+    public static final String CONFIG_OF_CURRENT_DATA_TABLE_VERSION = "pinot.server.instance.currentDataTableVersion";
+    public static final int DEFAULT_CURRENT_DATA_TABLE_VERSION = 3;
```

Comment:
I suggest changing the `private static int CURRENT_VERSION = VERSION_3` defined in DataTableBuilder to public and use that here instead of hardcoding the value 3

Reply:
+1

Reply:
This will cause a cynical cyclical dependency issue. So hardcode it as 3 at this moment